e2e/cypress/integration/system_console/reporting/site_statistics_spec.js

```diff
@@ -71,4 +77,131 @@ describe('System Console > Site Statistics', () => {
+
+    it('MM-T903 - Site Statistics > Deactivating a user increments the Daily and Monthly Active Users counts down', () => {
+        cy.apiInitSetup().then(({team, user}) => {
+            const testUser = user;
+            const testTeam = team;
+
+            // # Login as test user and visit town-square
+            cy.apiLogin(testUser);
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+
+            // # Wait two seconds then go to admin console
+            cy.wait(TIMEOUTS.TWO_SEC);
+            goToAdminConsole();
+
+            // # Go to system analytics
+            cy.get('#reporting\\/system_analytics').click();
+            cy.wait(TIMEOUTS.ONE_SEC);
+
+            let totalActiveUsersInitial;
+            let dailyActiveUsersInital;
+            let monthlyActiveUsersInital;
+            let totalActiveUsersFinal;
+            let dailyActiveUsersFinal;
+            let monthlyActiveUsersFinal;
+
+            // # Get the number and turn them into numbers
+            cy.findByTestId('totalActiveUsers').invoke('text').then((text) => {
+                totalActiveUsersInitial = parseInt(text, 10);
+                cy.findByTestId('dailyActiveUsers').invoke('text').then((text2) => {
+                    dailyActiveUsersInital = parseInt(text2, 10);
+                    cy.findByTestId('monthlyActiveUsers').invoke('text').then((text3) => {
+                        monthlyActiveUsersInital = parseInt(text3, 10);
+
+                        // # Deactivate user and relaod page and then wait 2 seconds
+                        cy.externalActivateUser(testUser.id, false);
+                        cy.reload();
+                        cy.wait(TIMEOUTS.TWO_SEC);
+
+                        // # Get the numbers required again
+                        cy.findByTestId('totalActiveUsers').invoke('text').then((text4) => {
+                            totalActiveUsersFinal = parseInt(text4, 10);
+
+                            cy.findByTestId('dailyActiveUsers').invoke('text').then((text5) => {
+                                dailyActiveUsersFinal = parseInt(text5, 10);
+
+                                cy.findByTestId('monthlyActiveUsers').invoke('text').then((text6) => {
+                                    monthlyActiveUsersFinal = parseInt(text6, 10);
+
+                                    // * Assert that the final number is the initial number minus one
+                                    expect(totalActiveUsersFinal).equal(totalActiveUsersInitial - 1);
+                                    expect(dailyActiveUsersFinal).equal(dailyActiveUsersInital - 1);
+                                    expect(monthlyActiveUsersFinal).equal(monthlyActiveUsersInital - 1);
+                                });
+                            });
+                        });
+                    });
+                });
+            });
+        });
+    });
+
+    it('MM-T905 - Site Statistics card labels in different languages', () => {
```

Comment:
An after block has to be added in this file to reset the sysadmin's language back to English. All the subsequent tests dependent on sysadmin login will start to fail after this test is executed.

Reply:
Gotcha! Good call ❤️

Reply:
Done!